Python-Import-Statements.md

```diff
@@ -0,0 +1,83 @@
+# Python Import Statements
+While learning programming and while reading some resources you'd have come across this word 'abstraction' which simply put means to reduce and reuse the code as much as possible.
```

Comment:
~~While learning programming and while reading~~ While learning programming and reading
~~which simply put means~~ which simply means